docs/rails-5-upgrade.md

```diff
@@ -0,0 +1,136 @@
+# Rails 5 Upgrade Steps
+
+1. First attempt to update to last 5.0 version of rails (5.0.7.2)
+  1. Attempted upgrade by updating rails version in Gemfile and running `bundle update rails` inside a /bin/bash session in the app Docker container.  Got bundler error: `genigames_connector was resolved to 0.0.5, which depends on rails (~> 4.2)`
+2. Attempted upgrading genigames_connector from 4.2 to 5.0
+  1. Created `rails5-support` branch from `spike-rails4-support`
+  2. Changed `s.add_dependency "rails", "~> 4.2"` to `s.add_dependency "rails", "~> 5"` in gemspec file and updated version in gem to 0.0.6
+  3. Added `- ..:/projects` to volumes section of app in docker-compose.yml (this points to my root Concord projects folder)
+  4. Ran the following in a rigse Docker bash session: `bundle config local.genigames_connector /projects/genigames-connector/`
+  5. Changed rigse Gemfile to `gem 'genigames_connector', git: 'https://github.com/concord-consortium/genigames-connector.git', branch: 'rails5-support'`
```

Comment:
clever 🧙

Reply:
I knew there had to be a better way to develop local gems.  It works with the caveat that the gem has to have a .git folder - the version of bundler we are using doesn't support the newer `path:` option which works with a plain folder.